appengine/pusher-chat/README.md

```diff
@@ -0,0 +1,49 @@
+# Pusher sample for Google App Engine
```

Comment:
As discussed, I will leave it here for now, but will submit a separate PR for J8